reliability-v2/config/reliability-netobserv.yaml

```diff
@@ -57,6 +59,8 @@ reliability:
         - func check_pods -n 2 # check pods in 2 namespaces 
         - func delete_project -n 2 # delete project in 2 namespaces
         - func verify_project_deletion -n 2 # verfy project deletion in 2 namespaces
+        - func check_flowcollector
+        - func check_netobserv_pods
```

Comment:
The 2 check functions should be put in only admin group, in which the tasks will be executed with kubeadmin user. The other groups users(testuser-xx) are idp users, they only have access to the project/ns that are created by themself, e.g. the test namespaces they created during the test.